CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [unreleased]
 
+### Changes
```

Comment:
This would e version 4.0 of the core? Since this is a breaking change (SQL table needs to be altered)

Reply:
Yeah, especially since it'll be a manual migration.